crates/model/src/fee_policy.rs

```diff
@@ -0,0 +1,34 @@
+use {
+    number::serialization::HexOrDecimalU256,
+    primitive_types::U256,
+    serde::{Deserialize, Serialize},
+    serde_with::serde_as,
+};
+
+#[serde_as]
+#[derive(PartialEq, Clone, Debug, Serialize, Deserialize)]
```

Comment:
Do these really need `Deserialize`?

Reply:
As long as `Trade` requires it at least for tests.

Reply:
it could be derived just for testing purposes

Reply:
It is used in `e2e` crate also. Used this approach: https://github.com/cowprotocol/services/pull/2809/commits/1b952902b6a17afac468564d20a81fdda07b2439